### PR TITLE
Rework menu and tabs layout

### PR DESCRIPTION
**Pull request by Lowentwickler**
- Fix logo on top with width of telemetry
- Rework tabs and menu layout to avoid visual bugs on different scaling
- Fix page layout and paddings for the edge of 1280px
- Change alignment for menu and print buttons
- Unify font-sizes

### CODE REVIEW

**Comment by Lowentwickler** [comment]
This is a 100% UI update, no code has changed.

File: src/fonts.css [hunk 1]
```diff
@@ -22,7 +16,7 @@ $text-black: #000000;
 
 html {
   line-height: 1.15;
-  font-size: 14pt;
+  font-size: 32pt;
```
Comment (ondratu) [inline src/fonts.css:19]: why not rem units too?

File: src/fonts.css [hunk 2]
```diff
@@ -91,26 +85,11 @@ select {
...
-  .txt-size-h1 {
-    font-size: $font-size-h1-desktop;
-  }
-  button > p {
-    font-size: $font-size-2-desktop;
-  }
-  nav {
-    font-size: $font-size-nav-desktop;
+  html {
+    font-size: 14pt;
```
Comment (ondratu) [inline src/fonts.css:93]: why not variable? Why not rem units?

File: src/styles.css [hunk 3]
```diff
@@ -468,7 +490,7 @@ nav .burger-icon.burger-open {
 }
 .temp-text {
   fill: #efefef;
-  font-size: 0.75rem;
+  font-size: 14pt;
```
Comment (ondratu) [inline src/styles.css:493]: why not variable? why not rem units?

File: src/styles.css [hunk 4]
```diff
@@ -483,7 +505,7 @@ nav .burger-icon.burger-open {
 }
 .temp-label-x,
 .temp-label-y {
-  font-size: 0.75rem;
+  font-size: 14pt;
```
Comment (ondratu) [inline src/styles.css:508]: why not variable? why not rem units?

File: src/styles.css [hunk 5]
```diff
@@ -515,7 +537,7 @@ nav .burger-icon.burger-open {
   stroke-linejoin: round;
 }
 .temp-line-text {
-  font-size: 1rem;
+  font-size: 14pt;
```
Comment (ondratu) [inline src/styles.css:540]: why not variable? why not rem units?

**Review by ondratu — CHANGES_REQUESTED** [review]

File: src/fonts.css (hunk 1 above)
Comment (Lowentwickler) [reply src/fonts.css:19]: rem units are relative to some root value. Root value must be set to `html`. All other sizes are relative to this one.

File: src/fonts.css (hunk 2 above)
Comment (Lowentwickler) [reply src/fonts.css:93]: Can be a variable of course. But there will be two new variables for two root values.

File: src/styles.css (hunk 3 above)
Comment (Lowentwickler) [reply src/styles.css:493]: Because this is for temperature SVG and it is being scaled on its own.

File: src/styles.css (hunk 4 above)
Comment (Lowentwickler) [reply src/styles.css:508]: Because this is for temperature SVG and it is being scaled on its own.

File: src/styles.css (hunk 5 above)
Comment (Lowentwickler) [reply src/styles.css:540]: Because this is for temperature SVG and it is being scaled on its own.

**Review by ondratu — APPROVED** [review]

**Pull request merged**